src/psyclone/domain/lfric/lfric_extract_driver_creator.py

```diff
@@ -962,7 +940,24 @@ def get_driver_as_string(self, nodes, input_list, output_list,
             print(str(err))
             return ""
 
-        return writer(file_container)
+        module_dependencies = self.collect_all_required_modules(file_container)
+        # Sort the modules by dependencies, i.e. start with modules
+        # that have no dependency. This is required for compilation, the
+        # compiler must have found any dependent modules before it can
+        # compile a module.
+        sorted_modules = ModuleManager.sort_modules(module_dependencies)
+
+        out = []
```

Comment:
Comment please. e.g. "Inline all required modules into the driver source file so that it is standalone."

Reply:
Done.